test/libsolidity/smtCheckerTests/try_catch/try_multiple_returned_values_with_tuple.sol

```diff
@@ -0,0 +1,26 @@
+pragma experimental SMTChecker;
+contract C {
+
+	struct S {
+		uint x;
+		int y;
+	}
+
+	function g() public pure returns (bool b, S memory s) {
+		b = true;
+		s.x = 42;
+		s.y = -1;
+	}
+
+	function f() public view {
+		bool success = false;
+		try this.g() returns (bool b, S memory s) {
+			success = true;
+			assert(b && s.x == 42 && s.y == -1);
+		} catch {
+		}
+		assert(success); // fails, not guaranteed that there will be no error
```

Comment:
Can you also add an assertion over `s` that holds?

Reply:
There already is an assertion on `s` that holds in the success clause and `s` is not visible outside that block, so I am not sure what you mean.